### PR TITLE
Improvements for Monitoring fencing devices

### PR DESCRIPTION
**Pull request by tahliar**
### PR creator: Description

Some improvements to the text in "Monitoring fencing devices".

### PR creator: Are there any relevant issues/feature requests?

* bsc#1219181
* jsc#DOCTEAM-1251

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP6
  - [x] 15 SP5
  - [x] 15 SP4
  - [x] 15 SP3
  - [x] 15 SP2
- SLE-HA 12
  - [x] 12 SP5

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by tahliar** [comment]
image

File: xml/ha_fencing.xml [hunk 1]
```diff
@@ -557,9 +546,12 @@ hostlist</screen>
   </para>
 
   <para>
-   Checking the status of fencing devices once every few hours should
-   be enough. The probability that a fencing operation needs to be
-   performed and the power switch fails is low.
+   The probability that a fencing operation needs to be performed and the
+   fencing device fails is low. For most devices a monitoring interval of
```
Comment (dariavladykina) [inline xml/ha_fencing.xml:550]:
```suggestion
   fencing device fails is low. For most devices, a monitoring interval of
```

File: xml/ha_fencing.xml [hunk 2]
```diff
@@ -557,9 +546,12 @@ hostlist</screen>
...
+   at least 1800 seconds (30 minutes) should suffice. The exact value depends
+   on the device and infrastructure. &stonith; SBD resources do not need a
+   monitor at all; see <xref linkend="sec-ha-fencing-special"/> and
```
Comment (dariavladykina) [inline xml/ha_fencing.xml:553]:
```suggestion
   monitor at all. See <xref linkend="sec-ha-fencing-special"/> and
```

**Review by dariavladykina — COMMENTED** [review]
Hi, some minorish suggestions :-) Thanks!

**Pull request merged**